magda-web-client/src/Components/RecordHandler.js

```diff
@@ -195,7 +196,24 @@ class RecordHandler extends React.Component {
                 const baseUrlDataset = `/dataset/${encodeURI(
                     this.props.match.params.datasetId
                 )}`;
-
+                if (this.props.dataset) {
```

Comment:
Doesn't putting this in `render` mean that it'll log duplicate views every time the state changes? In particular if someone comes along and puts say... a checkbox on the page that changes the state every time it's clicked.